experimental/devices/mfrc522/blockaccess.go

```diff
@@ -0,0 +1,73 @@
+package mfrc522
+
+import "fmt"
+
+// BlockAccess defines the block access bits.
```

Comment:
Can you expand on what this means? Same for SectorTrailerAccess.

Reply:
This is sort of hard to explain w/o reciting much of the datasheet.